dragonfly/block/stained_terracotta.go

```diff
@@ -0,0 +1,50 @@
+package block
+
+import (
+	"github.com/df-mc/dragonfly/dragonfly/block/colour"
+	"github.com/df-mc/dragonfly/dragonfly/item"
+	"github.com/df-mc/dragonfly/dragonfly/world"
+)
+
+// StainedTerracotta is a block formed from clay, with a hardness and blast resistance comparable to stone. In contrast
+// to Terracotta, t can be colored in the same 16 colors that wool can be dyed, but more dulled and earthen.
```

Comment:
- Typo: `t` should be `it`
- We generally use BE in dragonfly, so please use colour vs color.